tests/stats/mda8/test_mda8.py

```diff
@@ -49,6 +50,13 @@ def test_data(time, values) -> xr.DataArray:
             [np.nan] * 3,
             id="with-nans",
         ),
+        # https://github.com/metno/pyaerocom/issues/1323
+        pytest.param(
+            xr.date_range(start="2024-01-01 06:00:00", periods=30, freq="1h"),
+            np.arange(30),
+            [np.nan, 25.5],
```

Comment:
Not sure if the MDA8 for the 1st day is supposed to be NaN.
Similarly to my previous comment, there should be enough 8-hour means for the daily max.